hybrasyl/Objects/User.cs

```diff
@@ -104,6 +104,7 @@ public static string GetStorageKey(string name)
         public UserGroup Group { get; set; }
         [JsonProperty]
         public bool Dead { get; set; }
+        public bool IsCasting { get; set; }

```

Comment:
This should use the existing Conditions / Status stuff, in fact if I'm not mistaken there's a flag for it. Dead should either be vestigial or made to be vestigial.